appservice/src/intTest/groovy/nu/yona/server/UserTest.groovy

```diff
@@ -272,6 +275,8 @@ class UserTest extends AbstractAppServiceIntegrationTest
 		assert user.firstName == "John"
 		assert user.lastName == "Doe"
 		assert user.mobileNumber == "+${timestamp}"
+		assertEquals(user.creationTime, ZonedDateTime.now())
```

Comment:
Please use ``YonaServer.now`` instead of ``ZonedDateTime.now()``. This is a relatively new API to centralize the creation of time stamps in the Europe/Amsterdam time zone.